charts/kyverno-aws-adapter/README.md.gotmpl

```diff
@@ -15,11 +15,18 @@ You’ll need an [EKS](https://aws.amazon.com/eks/) cluster to run against.
 ### Running on the EKS cluster
 
 1. Make sure that you have configured an [IAM role for the service account](#IAM-Role-for-Service-Account) `kyverno-aws-adapter-sa` in your desired namespace (configured in `values.yaml`) and specified the role's ARN in the `roleArn` field inside `values.yaml` file.
-2. Install the Helm chart after making any necessary changes to `charts/kyverno-aws-adapter/values.yaml`
+2. Add the Kyverno AWS Adapter Helm repository.
+
+```console
+helm repo add kyverno-aws-adapter https://nirmata.github.io/kyverno-aws-adapter/
```

Comment:
This repo still not existent.  Chart release or incorrect path. Need to fix.

Reply:
It now exists, first valid release is being cut 🤞